consensus/consensus.md

```diff
@@ -277,6 +283,8 @@ The gas of a transaction is the sum of:
 | Channel offchain       |            0   |     0 |
 | Contract create        | `5 * BaseGas`  | Proportional to the byte size of the transaction, specifically: `byte_size(ContractCreateTx) * GasPerByte`. It also includes gas for contract execution. |
 | Contract call          | `30 * BaseGas` | Proportional to the byte size of the transaction, specifically: `byte_size(ContractCallTx) * GasPerByte`. It also includes gas for contract execution. |
+| GA Attach              | `5 * BaseGas`  | Proportional to the byte size of the transaction, specifically: `byte_size(GAAttachTx) * GasPerByte`. It also includes gas for execution of the init function. |
+| GA Meta                | `5 * BaseGas`  | Proportional to the byte size of the transaction, specifically: `byte_size(GAMetaTx) * GasPerByte`. It also includes gas for execution of the authentication function + gas for the wrapped transaction. |
```

Comment:
I think that this is taken care of by https://github.com/aeternity/aeternity/pull/2362 when it comes to actual charging. But the `gas_limit` will indeed make use be a bit pessimistic when it comes to picking what candidate transactions may fit in a microblock. This could be made more precise but with some additional complexity.

Reply:
Tracked https://www.pivotaltracker.com/story/show/166018387